frontend/src/css/NotificationWall.styl

```diff
@@ -43,6 +43,7 @@
         padding-left 5px
         padding-right 10px
         font-size fontSizeSmall
+        word-break break-word   
```

Comment:
keep in mind that the value "break-word" is deprecated by MDN (see : https://developer.mozilla.org/en-US/docs/Web/CSS/word-break)

but since we already use it in other places in tracim, it's ok to keep it